modules/ROOT/pages/reproducible-builds.adoc

```diff
@@ -7,5 +7,5 @@ You should be able to build a particular version of your Mule project at any poi
 
```

Comment:
edit of HIDDEN TEXT

= Reproducible Builds
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
You should be able to build a particular version of your Mule project at any point in time. This facilitates maintenance, because the versions of your projects in production are not always the latest ones.